src/chainparams.cpp

```diff
@@ -131,9 +131,9 @@ class CMainParams : public CChainParams
         bnProofOfWorkLimit = ~uint256(0) >> 20; // PIVX starting difficulty is 1 / 2^12
         nSubsidyHalvingInterval = 210000;
         nMaxReorganizationDepth = 100;
-        nEnforceBlockUpgradeMajority = 750;
-        nRejectBlockOutdatedMajority = 950;
-        nToCheckBlockUpgradeMajority = 1000;
+        nEnforceBlockUpgradeMajority = 1080;
+        nRejectBlockOutdatedMajority = 1368;
+        nToCheckBlockUpgradeMajority = 1440;
```

Comment:
After thinking about this for a while, I'm pretty sure that only looking at the last day's worth of blocks is much too short of a sample size, and leaves a **VERY** short window for people to update before enforcement kicks in.

Looking at the last 7 days worth of blocks (10080) probably makes more sense, with the thresholds (75%/95%) adjusted accordingly.